sample-apps/javascript-sample-app/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "@opentelemetry/api": "^1.2.0",
+    "@opentelemetry/exporter-collector-grpc": "^0.25.0",
```

Comment:
This Exporter is deprecated from a year ago. New exporters need to be used to be in compatible with new api/deps

 * https://www.npmjs.com/package/@opentelemetry/exporter-collector-grpc

Reply:
Thanks. Updated.